### PR TITLE
bootstrap: remove factory from cluster up command

### PR DESCRIPTION
**Pull request by mfojtik**
@deads2k not needed, we can get the default client config directly

(only last commit, others are part of https://github.com/openshift/origin/pull/19208
/cc @soltysh

### CODE REVIEW

File: pkg/oc/bootstrap/docker/up.go [hunk 1]
```diff
@@ -196,9 +196,8 @@ type ClusterUpConfig struct {
 	dockerHelper    *dockerhelper.Helper
 	hostHelper      *host.HostHelper
 	openshiftHelper *openshift.Helper
-	factory         *clientcmd.Factory
-	originalFactory *clientcmd.Factory
 	command         *cobra.Command
+	existingConfig  clientcmdapi.Config
```
Comment (deads2k) [inline pkg/oc/bootstrap/docker/up.go]: Better name please

File: pkg/oc/bootstrap/docker/up.go [hunk 2]
```diff
@@ -504,8 +513,6 @@ func (c *ClusterUpConfig) Start(out io.Writer) error {
 	webConsoleInstall := &web_console.WebConsoleComponentOptions{
 		PublicConsoleURL: fmt.Sprintf("https://%s:8443/console", c.GetPublicHostName()),
 		PublicMasterURL:  fmt.Sprintf("https://%s:8443", c.GetPublicHostName()),
-		PublicLoggingURL: fmt.Sprintf("https://%s", openshift.LoggingHost(c.RoutingSuffix)),
```
Comment (deads2k) [inline pkg/oc/bootstrap/docker/up.go]: Oh, this is awkward.  @spadgett I delete the option and left this maybe?

**Comment by mfojtik** [comment]
@deads2k updated, will rebase after context merges

**Pull request merged**